components/sql/src/main/scala/pl/touk/nussknacker/sql/service/DatabaseLookupEnricher.scala

```diff
@@ -136,4 +137,11 @@ class DatabaseLookupEnricher(dBPoolConfig: DBPoolConfig, dbMetaDataProvider: DbM
       keyColumnParamStep(context, dependencies) orElse
       finalStep(context, dependencies)
 
+  private def extractOrEvaluate(params: Map[String, Any], paramName: String, context: Context) = {
```

Comment:
WDYT about wrapping these `params` in `GenerincNodeTransformation.implementation` with class with such a method? Another nice method would be method evaluating all params at one time. Maybe touching this API is the good time for that?